src/Microsoft.ML.Data/TrainCatalog.cs

```diff
@@ -485,22 +485,22 @@ internal MulticlassClassificationTrainers(MulticlassClassificationCatalog catalo
         /// <param name="label">The name of the label column in <paramref name="data"/>.</param>
         /// <param name="score">The name of the score column in <paramref name="data"/>.</param>
         /// <param name="predictedLabel">The name of the predicted label column in <paramref name="data"/>.</param>
-        /// <param name="topK">If given a positive value, the <see cref="MultiClassClassifierMetrics.TopKAccuracy"/> will be filled with
+        /// <param name="topK">If given a positive value, the <see cref="MultiClassClassificationMetrics.TopKAccuracy"/> will be filled with
```

Comment:
>topK [](start = 25, length = 4)

Not sure if it's topK or just K.

Reply:
Yes I think this is k, but I almost prefer it like this.  Just one letter seems worst.

---
In reply to: [264771658](https://github.com/dotnet/machinelearning/pull/2919#discussion_r264771658) [](ancestors = 264771658)